test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -1007,6 +1008,59 @@ TEST_F(nntrainer_Conv2DLayer, backwarding_01_p) {
   matchOutput(bias_grad, "tc_conv2d_1_goldenBiasGrad.out");
 }
 
+TEST_F(nntrainer_Conv2DLayer, backwarding_0r_p) {
```

Comment:
How about add this unittest in a different PR.

Reply:
also, updating the unittest_layers.tar.gz with integers still needs more testing with loss. Lets do that at once in a separate PR.

Reply:
Further corresponding gen_input.py also needs proper changes to be able to produce this unittest data.
If you want to add it yourself, please refer to https://github.com/kparichay/nntrainer/tree/conv_forwarding_mismatch
, or I will add this unittest after this PR is merged.

Reply:
I will.